src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -211,9 +200,13 @@ public TimeSpan TimeToLive
 
         // TODO: Calculate the size of the properties and body
         /// <summary>
-        /// Gets the total size of the message in bytes.
+        /// Gets the total size of the message body in bytes.
```

Comment:
Rather than having a property called `Size` that represents body size only and later introduce another property for the total size, call it what is 🙂 - ` BodySize` and add `Size` later.